packages/tailwindcss/src/css-functions.test.ts

```diff
@@ -757,6 +757,101 @@ describe('theme function', () => {
   })
 })
 
+describe('screen function', () => {
+  describe('in candidates', () => {
+    test('[@media(screen(sm))]:flex', async () => {
+      expect(
+        await compileCss(
+          css`
+            @tailwind utilities;
+            @theme {
+              --breakpoint-sm: 48rem;
+            }
+          `,
+          ['[@media_screen(sm)]:flex'],
+        ),
+      ).toMatchInlineSnapshot(`
+        "@media (width >= 48rem) {
+          .\\[\\@media_screen\\(sm\\)\\]\\:flex {
+            display: flex;
+          }
+        }
+
+        :root {
+          --breakpoint-sm: 48rem;
+        }"
+      `)
+    })
+  })
+
+  describe('in @media queries', () => {
+    test('@media screen(sm)', async () => {
+      expect(
+        await compileCss(css`
+          @theme {
+            --breakpoint-sm: 48rem;
+          }
+          @media screen(sm) {
+            .foo {
+              display: flex;
+            }
+          }
+        `),
+      ).toMatchInlineSnapshot(`
+        ":root {
+          --breakpoint-sm: 48rem;
+        }
+
+        @media (width >= 48rem) {
+          .foo {
+            display: flex;
+          }
+        }"
+      `)
+    })
+
+    test('@media screen(advanced)', async () => {
```

Comment:
None of these are going to pass at least until we do the complex screen config stuff in #14415. Probably will still need more work after that.